pkg/api/api.go

```diff
@@ -293,6 +293,8 @@ func (hs *HttpServer) registerRoutes() {
 		apiRoute.Group("/alerts", func(alertsRoute RouteRegister) {
 			alertsRoute.Post("/test", bind(dtos.AlertTestCommand{}), wrap(AlertTest))
 			alertsRoute.Post("/:alertId/pause", reqEditorRole, bind(dtos.PauseAlertCommand{}), wrap(PauseAlert))
+			alertsRoute.Post("/pause/by-id/:alertId", reqEditorRole, bind(dtos.PauseAlertCommand{}), wrap(PauseAlert))
+			alertsRoute.Post("/pause/by-dashboard/:dashboardId", reqEditorRole, bind(dtos.PauseDashboardAlertsCommand{}), wrap(PauseDashboardAlerts))
```

Comment:
I'm still not sure about this path. (I know it was my suggestion.. )

We want to support pausing by dashboardid, datasource and dashboard folder (unreleased feature).
Having one endpoint for each seems messy. So I think we can combine them all into one endpoint and get the dashboardid/datasourceid/folderid from the json body instead.

`POST /alerts/pause` should require at least one of those fields to be set and pause alert rules according to the provided fields. 

@torkelo any thought on this? So far this is the only time we update entities based on foreign Id. I'm not 100% sure how we want to do that. 

All other changes look good. Thank you for being so patient with my feedback :)

Reply:
I suppose specifying the id in the json body also makes it possible to send a list of ids and do a bulk update. This isn't possible by extracting from a URL param right?

Reply:
@Jonnymcc yea that would also be a good addition.